src/__tests__/routingTable.model.test.ts

```diff
@@ -0,0 +1,104 @@
+import { Guid } from "guid-typescript";
+import { PeerType } from "../enums";
+import { PeerNode, RoutingTable } from "../models";
+import { TestIps } from "./testAddress";
+
+let routingTable: RoutingTable;
+
+beforeEach(() => {
+    routingTable = new RoutingTable();
+});
+
+describe("Creating routingTable model", () => {
+    it("Should match the defined routingTable format & properties values", () => {
+        expect(routingTable).toBeDefined();
+        expect(routingTable.peers).toBeDefined();
+        expect(routingTable.peers).toBeInstanceOf(Map);
+    });
+});
+
+describe("Adding/removing a peer", () => {
+    it("Should add/remove the peer to/from the peers map", () => {
+        const peerGuid = Guid.create().toString();
+        const ip = TestIps.TEST_1;
+
+        routingTable.addPeer(peerGuid, ip, PeerType.VALIDATOR);
+        const peerNode = routingTable.peers.get(peerGuid);
+
+        expect(peerNode).not.toBeUndefined();
```

Comment:
Please change "expect(peerNode).not.toBeUndefined();" to:

expect(peerNode).toBeDefined();